easybuild/easyconfigs/v/vawk/vawk-0.0.1.eb

```
@@ -0,0 +1,43 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'vawk'
+version = '0.0.1'
+
+homepage = 'https://github.com/cc2qe/vawk'
+description = """An awk-like VCF parser 
+"""
+
+toolchain = SYSTEM  # or = {'name': 'foss', 'version': '2019b'}
```

This easyconfig should use the `GCCcore/8.3.0` toolchian because it depends on `Python/2.7.16-GCCcore-8.3.0`
```suggestion
toolchain = {'name': 'GCCcore', 'version': '2019b'}
```